### ISSUE TITLE
Some discrepancies from the paper

### ISSUE THREAD

**Issue by ericjang** [eambutu/snail-pytorch#3]
For the Mini-ImageNet ResNet-based encoder, I believe the authors use mean pooling instead of max pooling, as done by this implementation. I am currently unable to reproduce the mini-imagenet results with your codebase (even train accuracy does not go up), and am hunting for discrepancies :)

**Comment by eambutu** [comment]
Awesome thanks so much! I hadn't achieved parity with MiniImagenet yet when I'd stopped working on this, so I appreciate the help! I finally have access to compute resources again, so I might try again this weekend.

One thing that might be worth a shot is trying loading/processing MiniImagenet in a different way: in other meta-learning repos using Miniimagenet, they download the images from the link in (https://github.com/dragen1860/LearningToCompare-Pytorch/issues/4) and do some other normalization (see: https://github.com/dragen1860/MAML-Pytorch/blob/master/MiniImagenet.py#L55).

**Comment by suoql** [comment]
Hey guys, thanks for sharing the suggestions! I added image transformation based on your dataloader. But the train accuracy still cannot go up. Are you guys able to reproduce the results on miniImagenet so far?
